### DISCUSSION TITLE
Question about aimbot code

### DISCUSSION THREAD

**Discussion by stratxgy** [ttwizz/Open-Aimbot#78] [Q&A] [answered]
Hi! I was wondering if it's called open aimbot because you are allowed to use code in your own project or because it's open source. Basicially wondering if I can use this code. Thanks.

**Comment by ttwizz** [comment, marked as answer]
> Hi! I was wondering if it's called open aimbot because you are allowed to use code in your own project or because it's open source. Basicially wondering if I can use this code. Thanks.

You can use this code in your own purposes, but don't forget to leave credits, as the source is licensed.
